apps/docs/docs/reference/ui/molecules/CheckboxField.mdx

```diff
@@ -0,0 +1,82 @@
+---
+description: CheckboxField grouped a Checkbox input and its corresponding Label.
+sidebar_custom_props:
+  image: https://vtexhelp.vtexassets.com/assets/docs/src/Checkbox___5dfb3d931c093ee7ebfac8bc388e68b5.png
```

Comment:
@carolinamenezes carol, can you pls replace this image later? It's the same idea of RadioField. `Checkbox` it's just the input, and the `CheckboxField` (`Checkbox` + `Label`)